TabGroup/TabGroup.module.less

```diff
@@ -20,7 +20,6 @@
 				color: @agate-panels-tab-color;
 				// background-color: @agate-panels-tab-bg-color;
 				// background-image: @agate-panels-tab-bg-image;
-				border-left: @agate-panels-tab-separator-border-width solid @agate-panels-tab-separator-border-color;
```

Comment:
For `vertical`, should we add `border-bottom`? WDYT?

Reply:
I agree